src/main/resources/META-INF/squbs-meta.conf

```diff
@@ -13,3 +13,8 @@ squbs-actors = [
     name = sample
   }
 ]
+squbs-extensions = [
```

Comment:
I am open either way. Unlike rocksqubs, the unicomplex lifecycle events are not observed ... so the kill will terminate app regardless of lifecylce. I can see both points.